packages/@react-spectrum/dnd/src/useDragAndDrop.ts

```diff
@@ -58,15 +58,17 @@ interface DropHooks {
 
 export interface DragAndDropHooks {
   /** Drag and drop hooks for the collection element.  */
-  dragAndDropHooks: DragHooks & DropHooks & {isVirtualDragging?: () => boolean}
+  dragAndDropHooks: DragHooks & DropHooks & {isVirtualDragging?: () => boolean, renderPreview?: (keys: Set<Key>, draggedKey: Key) => JSX.Element}
```

Comment:
what do you think about providing `draggedItem` instead of `draggedKey` since we can get the collection item's information for the user anyways? Not the biggest deal since the user can use the `draggedKey` to get the item's information themselves but just a though I had since we grab that info for the spectrum drag preview anyways

Reply:
Hmm... I would probably lean towards just key for consistency with our other APIs. I'll ask the team to see if there is any consensus.

Reply:
Looks like we will go for `key` for now, then take another look before release.